src/makeInfinityAdder.js

```diff
@@ -4,7 +4,21 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(number) {
+    if (number === undefined) {
```

Comment:
Consider using `typeof number === 'undefined'` instead of `number === undefined` to avoid potential issues if `undefined` is reassigned.